front/style/pages/_committee.scss

```diff
@@ -471,6 +471,10 @@
         }
     }
 
+    .box-right {
```

Comment:
can we add this general rule to `_layout.scss` file with `float-right` as the name?

Reply:
We had the same thought but @thomasbisignani told me there is a pull-righ / pull-left from font awesome css we should use as it already exists (next commit).

Reply:
hm, do we use FA?

Reply:
yes sir => https://github.com/EnMarche/en-marche.fr/blob/master/templates/base.html.twig#L97

Reply:
lol, thanks, I don't find it in package.json